auction-house/program/src/cancel/mod.rs

```diff
@@ -10,7 +10,7 @@ pub struct Cancel<'info> {
     /// CHECK: Verified in cancel_logic.
     /// User wallet account.
     #[account(mut)]
-    pub wallet: UncheckedAccount<'info>,
+    pub wallet: Signer<'info>,
```

Comment:
This is ok for now but we aren't going to be able to keep the changes from unchecked to signer. Will need to do invoke signed call without anchor CPI in order to keep the original signers.

Reply:
True. The problem is, we gotta find out if anchor inserts any other instruction data before what's ours (tag of instruction followed by price and token_size)